mvc/src/main/java/com/interface21/webmvc/servlet/view/MvcViewResolver.java

```diff
@@ -0,0 +1,21 @@
+package com.interface21.webmvc.servlet.view;
+
+import com.interface21.webmvc.servlet.View;
+import com.interface21.webmvc.servlet.view.type.JspView;
+
+public class MvcViewResolver implements ViewResolver {
+
+    private static final String EXTENSION_SEPARATOR = ".";
+    private static final String JSP_EXTENSION = ".jsp";
+
+    @Override
+    public View resolveViewName(String viewName) {
+        int extensionSeparatorIndex = viewName.lastIndexOf(EXTENSION_SEPARATOR);
+        String extension = viewName.substring(extensionSeparatorIndex);
+
+        if(extension.equals(JSP_EXTENSION)) {
+            return new JspView(viewName);
+        }
```

Comment:
```suggestion
        if (viewName.endsWith(JSP_EXTENSION)) {
            return new JspView(viewName);
        }
```
endsWith가 아니라 substring으로 분리한 이유가 있나요?ㅎㅎ